pandora-client-web/src/components/Eula/privacyPolicy.tsx

```diff
@@ -75,8 +75,10 @@ export function PrivacyPolicyContent(): ReactElement {
 				</li>
 			</ul>
 			<p>We do not share Your personal information with third parties.</p>
-			<h3>How do we store the data?</h3>
-			<p>We will retain the aforementioned data potentially indefinitely so we can fulfill the purposes described in this Privacy Policy; typically for the lifetime of Your Pandora account.</p>
+			<h3>How do We store the data?</h3>
+			<p>We will retain the aforementioned data potentially indefinitely so We can fulfill the purposes described in this Privacy Policy; typically for the lifetime of Your Pandora account.</p>
+			<p>If You do not log into Your Pandora account during a full calendar year, We will delete Your Pandora account and all the data stored in it during the following year.</p>
+			<p>In the case of Direct/Private Messages to other users, We will retain the encrypted data for a maximum of one year.</p>
```

Comment:
Why is this a different time span (less than a year) compared to the account (more than a year)?

Reply:
Wasn't there the rumor that the GDPR demands that it is deleted after 1 year?
I tried looking into it a bit, but did not find any specific section that states this, but some sites seem to interpret it like that.

Reply:
no messaging sites that I am aware of delete conversation messages older then 1 year so we can be quite sure this is not a GDPR requirement

Reply:
Reverted the new text and opened issue:
https://github.com/Project-Pandora-Game/pandora/issues/610